src/tailwind.js

```diff
@@ -86,9 +94,15 @@ module.exports = {
     },
   },
   variants: {
-    backgroundColor: ["responsive", "hover", "focus", "active"],
-    textColor: ["responsive", "hover", "focus", "active"],
-    borderRadius: ["responsive", "hover", "focus", "active"],
+    backgroundColor: ["responsive", "hover", "focus"],
+    textColor: ["responsive", "hover", "focus"],
+    borderRadius: ["responsive", "hover", "focus"],
```

Comment:
remove, these are already defaults. don't have to rewrite here

Reply:
done.